### ISSUE TITLE
email address?

### ISSUE THREAD

**Issue by maelle** [ropensci-org/pkgreviewr#52]
Does the package really use the user's email address? I don't think so?

In the README it could be advised to either do the git settings your recommend, or to set the `GITHUB_USERNAME` environment variable (that'll still be needed if the email address isn't public on GitHub anyway), or maybe it's just easier to refer to `whoami` docs?

**Comment by annakrystalli** [comment]
Yes adding this option to the docs is a good idea. Will address when I do the README/vignette overhaul
